meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -38,5 +40,14 @@ do_install_append_mender-systemd() {
         -C ${S} \
         DESTDIR=${D} \
         install-systemd
+    install -d ${D}/${localstatedir}/lib/mender-monitor
```

Comment:
This is the `mender-systemd` block. The install block is above.

Reply:
wow. I mean wow. I lost it completely :) sorry. fixed. copy-paste mistake.

Reply:
my mind is already in the next task.